general/templates/general/base.html

```diff
@@ -44,7 +44,7 @@
 
     <body class="d-flex flex-column min-vh-100">
         {% if config.get('YODA_ENVIRONMENT') != "production" %}{% include 'environment.html' %}{% endif %}
-        {% if config.get('banner_enabled') %}{% include 'banner.html' %}{% endif %}
+        {% if config.get('banner').banner_enabled %}{% include 'banner.html' %}{% endif %}
```

Comment:
Sidenote: add this to base-bare.html as well sometime in future

Reply:
Suggestion adopted.